game_runner.rb

```diff
@@ -0,0 +1,23 @@
+class GameRunner
+  BOARD_TOP = " __ __ __\n"
+
+  def self.run(*args)
+    GameRunner.new.run(*args)
+  end
+
+  def run
+    puts empty_board(3)
```

Comment:
Do you anticipate the board supporting different number of rows?

Reply:
😬 Good call out. No.